relay-server/src/services/buffer/envelope_buffer/mod.rs

```diff
@@ -315,8 +338,19 @@ where
     /// Returns `true` if at least one priority was changed.
     pub fn mark_ready(&mut self, project: &ProjectKey, is_ready: bool) -> bool {
         let mut changed = false;
-        if let Some(project_key_pair) = self.stacks_by_project.get(project) {
-            for project_key_pair in project_key_pair {
+        if let Some(project_key_pairs) = self.stacks_by_project.get(project) {
+            for project_key_pair in project_key_pairs {
+                if let Some((
+                    QueueItem {
+                        key: _,
+                        value: _,
+                        last_update,
+                    },
+                    _,
+                )) = self.priority_queue.get_mut(project_key_pair)
```

Comment:
Could write this as
```suggestion
                if let Some((QueueItem { last_update, .. }, _)) =
                    self.priority_queue.get_mut(project_key_pair)

```
but it's also fine as is.

Reply:
Oh nice, I forgot rust had this operator.